ads/_ping_.js

```diff
@@ -47,6 +47,12 @@ export function _ping_(global, data) {
     } else {
       context.renderStart();
     }
+    window.context.observeIntersection(function(changes) {
+      changes.forEach(function(c) {
+        dev().info('Intersection: ' +
```

Comment:
one suggestion would be indicating 'Intersection (width*height)` here to make it clearer.